### PR TITLE
xfce-extra/xfce4-whiskermenu-plugin: version bump to 1.5.3

### PR DESCRIPTION
**Pull request by tietzc**
As previous maintainer hasufell is retiring, I would like to proxy-maintain xfce-extra/xfce4-whiskermenu-plugin. I have already bumped this to version 1.5.2 in commit 09e5b010a4eacb51b862dcb2b33409780651a106. Moreover, I keep this up to date locally anyway, so why not share it.

Ebuild Changes:

* bump to EAPI 6

* minor fixes (corrected alphabetical order in line inherit, https instead of http)

### CODE REVIEW

**Comment by tietzc** [comment]
QA check failures are obviously not related to xfce-extra/xfce4-whiskermenu-plugin.

**Comment by tietzc** [comment]
@proxy-maint

File: xfce-extra/xfce4-whiskermenu-plugin/xfce4-whiskermenu-plugin-1.5.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	virtual/pkgconfig
+	sys-devel/gettext"
+
+src_prepare() {
+	local i
+	cd po || die
+	if [[ -n "${LINGUAS+x}" ]] ; then
+		for i in *.po ; do
+			einfo "removing ${i%.po} linguas"
+			has ${i%.po} ${LINGUAS} || { rm ${i} || die ; }
```
Comment (monsieurp) [inline xfce-extra/xfce4-whiskermenu-plugin/xfce4-whiskermenu-plugin-1.5.3.ebuild]: Is it just me or this line is trying to be too smart? :mag_right:

Comment (monsieurp) [inline xfce-extra/xfce4-whiskermenu-plugin/xfce4-whiskermenu-plugin-1.5.3.ebuild]: It turns out to be perfectly correct :+1:

**Comment by tietzc** [comment]
This may indeed appear a little too complex at first glance, but has actually been there for a long time and does exactly what it is supposed to do. In a most correct way like you say. Yet, I was irritated too when I first saw it.

Just for clarification: Am I correct to ignore the bot here? AFAIK the repo was simply borked when I created my branch. There is no connection to xfce-extra/whiskermenu.

CCing @perfinion from xfce-team who has already merged my last PR regarding this package.

**Comment by monsieurp** [comment]
> Just for clarification: Am I correct to ignore the bot here? AFAIK the repo was simply borked when I created my branch. There is no connection to xfce-extra/whiskermenu.

Very timely addendum to the wiki article: https://wiki.gentoo.org/wiki/Gentoo_Github#gentoo-repo-qa-bot_is_engaging_in_a_conversation_with_me_on_the_PR._I_don.27t_understand.
Short answer: yes, you can safely ignore this warning.

**Comment by tietzc** [comment]
Thank you very much. Hopefully the revised article will help reducing silly doubts and unnecessary questions concerning that matter.

**Comment by perfinion** [comment]
Hey,
This looks good and all. Just can you split the metadata.xml separate from the bump.
You should also have your name first in the metadata list since you're going to be maintaining. Otherwise all the bugs will go to xfce@. There is some accidental trailing whitespace.

Thanks!

**Comment by tietzc** [comment]
Suggestions applied by resetting the commit and pushing two new ones to my branch. Also fixed alphabetical order in DEPEND.

**Pull request closed without merging**